8.x/getting-started/documentation.md

```diff
@@ -2,4 +2,239 @@
 
 [[toc]]
 
-Larammerce is getting documented, as well as developed at high speed. So if you want to be on track, follow the guides.
\ No newline at end of file
+This documentation is going to define the steps of writing docs for larammerce.</br>
+larammerce documenting is with *markdown language* *<sup>[1](#1)</sup>* and you should also know about</br> *git commands* *<sup>[2](#2)</sup>* so you can easily start writing docs.
+
+### Starting point </br>
+First of all you have to **fork** `larammerce/larammerce-docs` repository which is already hosted on `github.com`.</br>
+Then you can modify any change on your own repository after you clone it on your local space.
+
+### quick preview
```

Comment:
Please consider removing the `How to work with git` from your document and send the readers to follow the instructions on the `Contribution Guide` page.